src/axom/primal/operators/intersection_volume.hpp

```diff
@@ -153,7 +157,7 @@ AXOM_HOST_DEVICE T intersection_volume(const Tetrahedron<T, 3>& tet1,
                                        double eps = 1.e-10,
                                        bool checkSign = false)
 {
-  return clip(tet1, tet2, eps, checkSign).volume();
+  return axom::utilities::abs(clip(tet1, tet2, eps, checkSign).volume());
```

Comment:
If this is using `Polyhedron::volume()` rather than `Polyhedron::signedVolume()`, it should always be positive, so this `abs` call is unnecessary.  (Or am I missing something?)